.chloggen/telemetrygen-traces-batch-size-options.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: telemetrygen
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adds options to configure the use of the batchspanprocessor and the size in MB of each trace for load testing.
```

Comment:
Could you name the options here? I would also talk about batching without making reference to the specific implementation using the batchspanprocessor